backend/src/scraper/consumer_reports_scraper.py

```diff
@@ -0,0 +1,54 @@
+from os import environ
+import requests
+from typing import Dict
+import difflib
+
+def scrape_expert_reviews(q: str, **kwargs) -> Dict:
+    short_title = ' '.join(q.split()[:5])
+    upc = kwargs.get('upc', '')
+    walmart_id = kwargs.get('walmart_id', '')
```

Comment:
Can we generalize this? Should be able to do something like:

```
def scrape_expert_reviews(q: str, id: str, id_field: str) -> Dict:

...

if str(id) == er.get(id_field):
```

Then in `app.py`:
```
if title:
        product_data.update(scrape_expert_reviews(title, product_data["basic_info"].get("upc"), "defaultUpc"))

if title:
        product_data.update(scrape_expert_reviews(title, product_id, "walmartId"))
```

Reply:
Also, don't feel like you need to implement this; it isn't worth the time that much lol

A lot of my code is probably pretty inefficient

Reply:
No, it's a good suggestion. I just updated it.